cql2/standard/clause_8_encodings.adoc

```diff
@@ -32,7 +32,7 @@ The list of CQL2 keywords includes:
 * "BETWEEN"
 * "CASEI"
 * "DATE"
-* "ENVELOPE"
+* "BBOX"
```

Comment:
The keywords list is alphabetical so BBOX should be moved up two lines.

Reply:
Thanks, fixed